sea.js

```diff
@@ -492,14 +509,16 @@
 
   ;USE(function(module){
     var shim = USE('./shim');
+    var S = USE('./settings');
     var sha256hash = USE('./sha256');
 
     const importGen = async (key, salt, opt) => {
       //const combo = shim.Buffer.concat([shim.Buffer.from(key, 'utf8'), salt || shim.random(8)]).toString('utf8') // old
       var opt = opt || {};
       const combo = key + (salt || shim.random(8)).toString('utf8'); // new
       const hash = shim.Buffer.from(await sha256hash(combo), 'binary')
-      return await shim.subtle.importKey('raw', new Uint8Array(hash), opt.name || 'AES-GCM', false, ['encrypt', 'decrypt'])
+      const jwkHash = S.keyTojwk(hash)
+      return await shim.subtle.importKey('jwk', jwkHash, {name:'AES-GCM'}, false, ['encrypt', 'decrypt'])
```

Comment:
Here also better use descriptive `jwk` props name, like this:
```
      const jwkKey = S.keyTojwk(hash)
      return await shim.subtle.importKey('jwk', jwkKey, { name: 'AES-GCM' }, false, [ 'encrypt', 'decrypt' ]);
```

Reply:
done